meteor-app/imports/ui/assessment2020/page-tabs.stories.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+// import { action } from '@storybook/addon-actions'
+
+import TabbedPages from './page-tabs'
+
+storiesOf('TabbedPages', module).add('TabbedPages', () => <TabbedPages />)
```

Comment:
In storybook this shows up as `TabbedPages/TabbedPages`. The first name should be a section, eg `Service`, and then other stories can live under this section

Reply:
I create a folder called `Service` and put all related components inside it.